### ISSUE TITLE
Inapropriate SEL label on shift.aug

### ISSUE THREAD

**Issue by samuraiii** [cern-it-sdc-id/puppet-lcgdm#12]
Hi,
threr is wrong SEL labeling on ${settings::libdir}/augeas/lenses/shift.aug in  manifests/base/config.pp which causes refresh of some services EVERY time puppet is run:
Before edit
```
Info: Retrieving pluginfacts
Info: Retrieving plugin
Notice: /File[/var/lib/puppet/lib/augeas/lenses/shift.aug]/seltype: seltype changed 'etc_t' to 'puppet_var_lib_t'
Info: Loading facts
Info: Caching catalog for golias100.farm.particle.cz
Info: Applying configuration version '1486888032'
Notice: /Stage[main]/Roles::Monitoring_nagios/File[/var/log/puppet/]/group: group changed 'puppet' to 'nagios'
Notice: /Stage[main]/Lcgdm::Base::Config/File[/var/lib/puppet/lib/augeas/lenses/shift.aug]/seltype: seltype changed 'puppet_var_lib_t' to 'etc_t'
Info: Class[Lcgdm::Base::Config]: Scheduling refresh of Class[Dmlite::Srm::Service]
Info: Class[Lcgdm::Base::Config]: Scheduling refresh of Class[Dmlite::Dav::Service]
Info: Class[Dmlite::Srm::Service]: Scheduling refresh of Service[srmv2.2]
Notice: /Stage[main]/Dmlite::Srm::Service/Service[srmv2.2]: Triggered 'refresh' from 1 events
Info: Class[Dmlite::Dav::Service]: Scheduling refresh of Service[httpd]
Notice: /Stage[main]/Dmlite::Dav::Service/Service[httpd]: Triggered 'refresh' from 1 events
Notice: Finished catalog run in 73.49 seconds
```
After edit
```
Info: Retrieving pluginfacts
Info: Retrieving plugin
Notice: /File[/var/lib/puppet/lib/augeas/lenses/shift.aug]/seltype: seltype changed 'etc_t' to 'puppet_var_lib_t'
Info: Loading facts
Info: Caching catalog for golias100.farm.particle.cz
Info: Applying configuration version '1486888278'
Notice: /Stage[main]/Roles::Monitoring_nagios/File[/var/log/puppet/]/group: group changed 'puppet' to 'nagios'
Notice: Finished catalog run in 65.19 seconds
```
See:
https://github.com/cern-it-sdc-id/puppet-lcgdm/pull/11

With wishes of nice day
S

**Comment by andrea-manzi** [comment]
Hello,
thanks a lot for reporting this, this is due a change i recently made to support puppet 4 and i didn't noticed this behaviour in my testbeds!
i will do some tests and then release the fix
thanks!
Andrea

**Comment by andrea-manzi** [comment]
i just released puppet-lcgdm 0.3.7 which fixes this issues
thanks!
Andrea
